subsys/bluetooth/mesh/net.c

```diff
@@ -44,7 +44,7 @@
 #define LOOPBACK_BUF_SUB(buf) (*(struct bt_mesh_subnet **)net_buf_user_data(buf))
 
 /* Seq limit after IV Update is triggered */
-#define IV_UPDATE_SEQ_LIMIT 8000000
```

Comment:
I saw the comment from @trond-snekvik mentioning the original being `0x8000000` but unless I'm missing something that's not the case?

Reply:
I think it is indeed decimal, in fact there is not much difference, `previous hex(8000000)=0x7A1200` vs `current 0x800000`.

Reply:
This value will be changed according to the needs of the appliance, so the initial value has no special significance.

Reply:
Yes, I agree.